modules/entity/src/state_selectors.ts

```diff
@@ -14,11 +14,10 @@ export function createSelectorsFactory<T>() {
     const selectAll = createSelector(
       selectIds,
       selectEntities,
-      (ids: T[], entities: Dictionary<T>): any =>
-        ids.map((id: any) => (entities as any)[id])
+      (ids, entities): any => ids.map((id: any) => (entities as any)[id])
```

Comment:
bewilderingly, even though the explicit typing of the projector parameters aligned perfectly with the inferred types, Typescript's brain breaks and thinks this matches the overload for returning `SelectorWithProps` when explicitly typed.

Reply:
you really should be able to explicitly declare the parameter types to a projector function, so I want this to be fixed, I just can't figure out what Typescript is mad about.